h2o-automl/src/main/java/ai/h2o/automl/ModelingStepsRegistry.java

```diff
@@ -82,6 +82,11 @@ public static void registerProvider(ModelingStepsProvider provider) {
                 orderedSteps.addAll(Arrays.asList(toAdd));
             }
         }
+        if (modelingPlan == AutoML.defaultModelingPlan)
```

Comment:
I'm working on fixing this and have better support for groups, starting with the tests.